### ISSUE TITLE
This code will not work on ie or Firefox

### ISSUE THREAD

**Issue by Kapana7** [FreeCodeCamp/FreeCodeCamp#4914]
Challenge [Waypoint: Style the HTML Body Element](http://www.freecodecamp.com/challenges/waypoint-style-the-html-body-element?solution=%3Cstyle%3E%0A%20%20body%20%7B%0A%20%20%20%20background-color%3A%20black%3B%0A%20%20%7D%0A%0A%3C%2Fstyle%3E%0A) has an issue.
User Agent is: <code>Mozilla/5.0 (Windows NT 6.3; WOW64; rv:42.0) Gecko/20100101 Firefox/42.0</code>.
Please describe how to reproduce this issue, and include links to screenshots if possible.

My code:
```html
<style>
  body {
    background-color: black;
  }

</style>

```

**Comment by raisedadead** [comment]
:+1:

**Comment by DDaems** [comment]
should be fixed in PR: #4903

**Comment by bugron** [comment]
@Kapana7 thanks for posting this issue!
@DDaems your PR is fixing another issue which is not related to this one. The origin of this issue goes to #4865. This will be fixed by #4884. Closing this issue.

>***
>Please review the [**Guidelines for Contributing**](https://github.com/FreeCodeCamp/FreeCodeCamp/blob/staging/CONTRIBUTING.md), thank you!.
>***
